src/components/profile.ts

```diff
@@ -0,0 +1,170 @@
+import { openDB } from './db';
+
+// since interfaces are for compile time there is no easy way to extract keys
+// so this userProfileKeys is needed
+export const userProfileKeys = [
```

Comment:
Is this being used anywhere?

Reply:
nope, you're right. is this enforced in ci? not sure since github actions didn't run for this branch for some reason